### PR TITLE
Danish translations

### PR DESCRIPTION
**Pull request by runephilosof**

### CODE REVIEW

File: app/locale/da.coffee [hunk 1]
```diff
@@ -136,12 +136,12 @@ module.exports = nativeDescription: "dansk", englishDescription: "Danish", trans
...
     get_started_subtitle: "Opsæt en klasse, tilføj dine elever, og overvåg deres fremgang når de lærer datalogi."
-    request_demo: "Anmod demo"
-#    request_quote: "Request a Quote"
+    request_demo: "Anmod om en demo"
+    request_quote: "Anmod om et tilbud"
     setup_a_class: "Opsæt klasse"
     have_an_account: "Har du allerede en konto?"
     logged_in_as: "Du er i øjeblikket logget ind som"
-    computer_science: "Datalogikurser for alle aldre" # {change}
+    computer_science: "Vores kurser dækker fra grundlæggende syntax til avancerede begreber" # {change} missing self-paced
```
Comment (runephilosof) [inline app/locale/da.coffee:144]: I know what self-paced means.
I just don't know how to express it well in danish.

File: app/locale/da.coffee [hunk 2]
```diff
@@ -279,55 +279,55 @@ module.exports = nativeDescription: "dansk", englishDescription: "Danish", trans
...
+    pass: "spring over"
+    return: "returnér"
+    then: "så"
+    do: "gør"
+    end: "slut"
+    function: "funktion"
+    def: "definér"
+    var: "variabel"
+    self: "selv"
+    hero: "helt (din kriger)"
```
Comment (runephilosof) [inline app/locale/da.coffee:300]: This means "hero (your warrior)".
Because `helt` is ambiguous.

File: app/locale/da.coffee [hunk 3]
```diff
@@ -279,55 +279,55 @@ module.exports = nativeDescription: "dansk", englishDescription: "Danish", trans
...
+    "+": "plus"
+    "-": "minus"
+    "+=": "læg sammen og tildel"
+    "-=": "træk fra og tildel"
+    True: "Sandt"
+    true: "sandt"
+    False: "Falsk"
+    false: "falsk"
+    undefined: "udefineret"
+    null: "null (udefineret)"
```
Comment (runephilosof) [inline app/locale/da.coffee:328]: Would it make sense to do the same in English, to explain that `null` and `nil` means undefined?
Or are `null` and `nil` common enough words in English?

File: app/locale/da.coffee [hunk 4]
```diff
@@ -1052,7 +1052,7 @@ module.exports = nativeDescription: "dansk", englishDescription: "Danish", trans
     action_radius: "Radius"
     action_duration: "Varighed"
     example: "Eksempel"
-    ex: "eks." # Abbreviation of "example"
+    ex: "fx" # Abbreviation of "example"
```
Comment (runephilosof) [inline app/locale/da.coffee:1055]: It is either `f.eks.` or `fx`, not `eks.`

**Comment by Bryukh** [comment]
Hi @runephilosof , thanks for your translations! Can you [sign the Contributor License Agreement](https://contributors.codecombat.com/cla) so we can merge this in?

**Comment by Bryukh** [comment]
Sorry closed as for CLA isn't signed. Feel free to reopen it when it's ready. Thanks!

**Pull request closed without merging**